app/controllers/idv/sessions_controller.rb

```diff
@@ -7,7 +7,16 @@ class SessionsController < ApplicationController
     def index
     end
 
+    def finance
+    end
+
     def create
+      self.idv_params = applicant_params.delete_if { |_key, value| value.blank? }
```

Comment:
Are all the fields optional or are any required? If some are required, then we should add presence validations and we wouldn't need to manipulate the hash. If all the fields are optional, what is the reason for removing blank values? What happens if a blank value is saved in the DB?